CHANGELOG.md

```diff
@@ -2,43 +2,40 @@
 
 ## New Features
 
+- PR #1993 Add CUDA-accelerated series aggregations: mean, var, std
 - PR #2111 IO Readers: Support memory buffer, file-like object, and URL inputs
 - PR #2012 Add `reindex()` to DataFrame and Series
 - PR #2097 Add GPU-accelerated AVRO reader
-- PR #2098 Align DataFrame and Series indices before executing binary ops
```

Comment:
This should stay in new features because it introduced new functionality as opposed to being transparent

Reply:
Can you say what the new feature is? The changelog entry is too vague for a new feature.